src/lib.rs

```diff
@@ -116,6 +116,7 @@ mod token;
 pub use event::{
     PollOpt,
     EventSet,
+    IoEvent,
```

Comment:
`Poll` is a public API that provides polling on only sockets (no timer / notifications). It returns `IoEvent`. I'm confused how Mio compiled before without having this exported.

However, I will track this as part of the api discussion